index.yaml

```diff
@@ -14,12 +14,12 @@ indexes:
 - kind: "Feature"
   properties:
   - name: "shipped_milestone"
+  - name: "shipped_android_milestone"
```

Comment:
Should this be updated?

Reply:
Sigh.  No, that is something that should be part of a different PR.  I have reverted the change via the GitHub UI.  GitHub still shows a diff, but it is now removing the line that I had added.  So, I think it is fine.  I tried pulling this PR branch to my workstation and it seems right.